client/src/Widgets/ActivityTable.tsx

```diff
@@ -0,0 +1,247 @@
+              activity.assignmentStatus !== undefined &&
+              activity.assignmentStatus !== "Unassigned"
+                ? activity.assignmentStatus
+                : "";
+
+            if (
+              activity.assignmentStatus === "Open" &&
+              activity.closeTime !== undefined
+            ) {
+              assignmentStatusString =
+                assignmentStatusString + " until " + activity.closeTime;
+            }
+
+            function saveUpdatedTitle(newTitle: string) {
+              if (newTitle !== activity.title && activity.id !== undefined) {
+                fetcher.submit(
+                  {
+                    _action: "update title",
+                    id: activity.id,
+                    cardTitle: newTitle,
+                    isFolder: Boolean(activity.isFolder),
+                  },
+                  { method: "post" },
+                );
+              }
+            }
+
+            return (
+              <Tr
+                key={activity.id + "_row"}
+                cursor="pointer"
+                _hover={{ backgroundColor: "#eeeeee" }}
+                borderBottom="2px solid gray"
+                onClick={() =>
+                  activity.cardLink ? navigate(activity.cardLink) : null
+                }
+              >
+                <Td p="0" m="0" width="20px">
+                  <Tooltip
+                    label={
+                      activity.isFolder
+                        ? activity.isPublic || activity.isShared
+                          ? "Folder / Public"
+                          : "Folder / Private"
+                        : activity.assignmentStatus === "Unassigned"
+                          ? "Activity / " + (activity.assignmentStatus ?? "")
+                          : "Assignment / " + (activity.assignmentStatus ?? "")
+                    }
+                  >
+                    <Box paddingRight="1em" m="0">
+                      <Icon
+                        as={
+                          activity.isFolder
+                            ? FaFolder
+                            : activity.assignmentStatus === "Unassigned"
+                              ? RiDraftFill
+                              : MdAssignment
+                        }
+                        color={
+                          activity.isFolder
+                            ? "#e6b800"
+                            : activity.assignmentStatus === "Unassigned"
+                              ? "#ff6600"
+                              : activity.assignmentStatus === "Open"
+                                ? "#009933"
+                                : "#cc3399"
+                        }
+                        boxSize={12}
+                        paddingLeft="1em"
+                      />
+                    </Box>
+                  </Tooltip>
+                </Td>
+                <Td>
+                  <HStack>
+                    <Editable
```

Comment:
When `editableTitle` is false, then the whole card should be clickable. You could probably swap out the `<Editable>` for just a `<Text>` in that case.